website/docs/components/form/layout/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,118 @@
+
+![An Error Alert component used within a Form appearing above all the fields serving as a form-level error message](/assets/components/form/layout/form-layout-error.png)
+
+## Form Section
+
+Inputs within a form are organized in Sections. Multiple Sections may be used to group content for a more scannable and organized experience. Inputs that belong under the same heading or are related to the same topic should be within a Section.
+
+### Form Section Header
+
+Similar to the Form Header, the Section Header contains styled and spaced text elements, and a space for custom content to be used as needed.
+
+![Section Header anatomy showing a Title, a Description, and a placeholder for additional content](/assets/components/form/layout/form-section-anatomy.png)
+
+### Form Section Multi Field Group
+
+The Section Multi Field Group organizes closely related form inputs in a single row, setting 16px between them, as opposed to 24px when stacked. This reinforces their relationship.
+
+![Section Multi Field Group example depicting spacing between grouped fields](/assets/components/form/layout/form-section-fieldgroup.png)
+
+In narrower screens, such as mobile devices, form inputs contained in a Section Multi Field Group automatically stack at the ['md' breakpoint](/foundations/breakpoints). 
+
+
+!!! Do
+
+Organize inputs together when they are closely related and beneficial to read together. No more than three inputs should be put within a single row.
+
+![Three credit card information fields arranged with one full width field followed by a row of two side-by-side equal width fields](/assets/components/form/layout/form-do-rows.png)
+
+!!!
+
+!!! Dont
+
+Place inputs in rows to save space or create columns within the form.
+
+![Three equal width fields for credit card information arranged in a single row](/assets/components/form/layout/form-dont-rows.png)
+
+!!!
+
+## Form Separator
+
+A Separator can be placed between Sections when a clearer visual distinction between Sections is needed.
+
+!!! Do
+
+Use a Separator to add space between Sections or before a new Section Header. 
+
+![Separator used to add extra visual differentiation between two Sections](/assets/components/form/layout/divider-spacing-do-1.png)
+
+!!!
+
+!!! Dont
+
+Don’t use a Separator between the last Section and the Button Set or form actions at the bottom of the Form content.
+
+![Separator being uses after all the content in a form](/assets/components/form/layout/divider-spacing-dont.png)
+
+!!!
+
+## Form Footer
+
+The Form Footer typically houses a Button or Button Set to allow users to submit form information. When there's more than one action, we recommend using the [Button Set](/patterns/form-patterns#button-sets) component for consistent spacing and layout.
+
+## Width and resizing behavior
+
+### Max-width
+
+Sections have a default max-width of 672px, establishing a consistent line length throughout applications.
```

Comment:
```suggestion
Sections have a default max-width of 672px, establishing a consistent line length throughout applications. It's possible to customize this max-width value if needed.
```

Reply:
We could also leave out this line: "It's possible to customize this max-width value if needed." since we already mention the max-width is a "default"